crates/klave/src/router.rs

```diff
@@ -7,3 +7,7 @@ pub fn add_user_query(query_function_name: &str) {
 pub fn add_user_transaction(transaction_function_name: &str) {
     sdk::add_user_transaction(&transaction_function_name);    
 }
+
+pub fn cancel_transaction() {
+    sdk::cancel_transaction();    
+}
```

Comment:
If this `adds` feature then the commit message should have been `feat(crate): ...`. Of course it is a judgement you're making, but good to keep in mind for next time.